scheduler/src/cook/rest/api.clj

```diff
@@ -28,6 +28,7 @@
             [compojure.core :refer [ANY GET POST routes]]
             [cook.compute-cluster :as cc]
             [cook.config :as config]
+            [cook.scheduler.constraints :as constraints]
```

Comment:
Let's keep the `require`s lexicographically ordered.

Reply:
by `cook.scheduler.constraints` or `constraints`? they seem to be roughly lexicographically ordered by the phrase after `:as`

Reply:
No. Intention is importts are leciconograpically. 
    cook.scheduler.constraints > cook.rest.cors

Should be put between cook.datomic and cook.scheduler.data-locality

Reply:
ah yeah, the part before the `:as`

Reply:
done